src/dfx/src/commands/ledger/create_canister.rs

```diff
@@ -47,6 +47,10 @@ pub struct CreateCanisterOpts {
     #[clap(long, validator(icpts_amount_validator))]
     max_fee: Option<String>,
 
+    /// Transaction timestamp, in nanoseconds, for use in controlling transaction-deduplication, default is system-time. // https://internetcomputer.org/docs/current/developer-docs/integrations/icrc-1/#transaction-deduplication-
+    #[clap(long)]
+    created_at_time: Option<String>,
```

Comment:
Would `Option<u64>` work here, and eliminate the need for the manual conversion?

Reply:
Yes Option<u64> works. Changed in this commit https://github.com/dfinity/sdk/pull/3018/commits/803dfdadc37efea6a9232fbec52f60d2af93878e .